src/overlays/actors/ovl_En_Zl3/z_en_zl3.c

```diff
@@ -15,421 +19,2827 @@ void EnZl3_Destroy(Actor* thisx, GlobalContext* globalCtx);
+}
+
+void func_80B590BC(EnZl3* this, GlobalContext* globalCtx) {
+    func_80B54DE0(this, globalCtx);
+    func_80B537E8(this);
+    func_80B536C4(this);
+    func_80B533FC(this, globalCtx);
+    func_80B5366C(this, globalCtx);
+    func_80B534CC(this);
+    EnZl3_FrameUpdateMatrix(this);
+    func_80B58624(this, globalCtx);
+}
+
+void func_80B5912C(EnZl3* this, GlobalContext* globalCtx) {
+    func_80B54DE0(this, globalCtx);
+    func_80B536C4(this);
+    func_80B533FC(this, globalCtx);
+    func_80B5366C(this, globalCtx);
+    func_80B56E38(this, globalCtx);
+    func_80B534CC(this);
+    EnZl3_FrameUpdateMatrix(this);
+    func_80B57CB4(this, globalCtx);
+    func_80B57D60(this, globalCtx);
+    func_80B58A1C(this, globalCtx);
+}
+
+void func_80B591BC(EnZl3* this, GlobalContext* globalCtx) {
+    func_80B54DE0(this, globalCtx);
+    func_80B536C4(this);
+    func_80B538B0(this);
+    func_80B533FC(this, globalCtx);
+    func_80B5366C(this, globalCtx);
+    func_80B534CC(this);
+    EnZl3_FrameUpdateMatrix(this);
+    func_80B58A50(this, globalCtx);
+}
+
+void func_80B5922C(EnZl3* this, GlobalContext* globalCtx) {
+    func_80B54DE0(this, globalCtx);
+    func_80B536C4(this);
+    func_80B57298(this);
+    Actor_SetHeight(&this->actor, 60.0f);
+    func_80B533FC(this, globalCtx);
+    func_80B5366C(this, globalCtx);
+    func_80B534CC(this);
+    EnZl3_FrameUpdateMatrix(this);
+    func_80B58AAC(this, globalCtx);
+}
+
+void func_80B592A8(EnZl3* this, GlobalContext* globalCtx) {
+    func_80B54DE0(this, globalCtx);
+    func_80B536C4(this);
+    func_80B57298(this);
+    Actor_SetHeight(&this->actor, 60.0f);
+    func_80B533FC(this, globalCtx);
+    func_80B5366C(this, globalCtx);
+    func_80B534CC(this);
+    func_80B57AAC(this, EnZl3_FrameUpdateMatrix(this), &D_06003D20);
+    func_80B56DEC(this);
+    func_80B58AAC(this, globalCtx);
+}
+
+void func_80B59340(EnZl3* this, GlobalContext* globalCtx) {
+    func_80B54DE0(this, globalCtx);
+    func_80B536C4(this);
+    func_80B57298(this);
+    Actor_SetHeight(&this->actor, 60.0f);
+    func_80B533FC(this, globalCtx);
+    func_80B5366C(this, globalCtx);
+    func_80B534CC(this);
+    func_80B57AAC(this, EnZl3_FrameUpdateMatrix(this), &D_06009FBC);
+    func_80B58AAC(this, globalCtx);
+}
+
+void func_80B593D0(EnZl3* this, GlobalContext* globalCtx) {
+    func_80B54DE0(this, globalCtx);
+    func_80B536C4(this);
+    func_80B57298(this);
+    func_80B5366C(this, globalCtx);
+    func_80B56E38(this, globalCtx);
+    Actor_SetHeight(&this->actor, 60.0f);
+    func_80B534CC(this);
+    EnZl3_FrameUpdateMatrix(this);
+    func_80B58C08(this, globalCtx);
+}
+
+s32 func_80B5944C(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, EnZl3* thisx,
+                  Gfx** gfx) {
+    if (limbIndex == 14) {
+        Mtx* mtx = Graph_Alloc(globalCtx->state.gfxCtx, sizeof(Mtx) * 7);
+        Vec3s* vec = &thisx->unk_3F8.unk_08;
+
+        gSPSegment(gfx[0]++, 0x0C, mtx);
```

Comment:
```suggestion
        gSPSegment((*gfx)++, 0x0C, mtx);
```

Reply:
This unfortunately causes a codegen change